src/ray/core_worker/task_manager.h

```diff
@@ -140,6 +140,8 @@ class TaskManager : public TaskFinisherInterface, public TaskResubmissionInterfa
   /// \param[in] error_type The type of the specific error.
   /// \param[in] status Optional status message.
   /// \param[in] ray_error_info The error information of a given error type.
+  /// Nullptr means that there's no error information.
+  /// TODO(sang): Remove nullptr case. Every error message should have metadata.
```

Comment:
How about we remove nullptr right away and instead add "TODO / unknown cause" in the call sites? Supporting special case missing information has bitten us in the past (e.g., raylet pid gets lost).

Reply:
This will be handled in the next clean up PR!